db/seeds.d/179_ui_notifications.rb

```diff
@@ -0,0 +1,10 @@
+blueprints = [
+  {
+    group: N_('Red Hat Insights'),
+    name: 'insights_satellite_hits',
+    message: N_('Satellite server has %{subject} recommendations by Red Hat'),
+    level: 'warning',
+  },
+]
+
+blueprints.each { |blueprint| UINotifications::Seed.new(blueprint).configure }
```

Comment:
nit: extra line here

Reply:
This is the line that actually creates the blueprint objects in the DB. The lines above are just definitions of the `blueprints` attributes array.

Reply:
No newline at end of file